src/tests/unit/tests/scanner/__snapshots__/get-rule-inclusions.test.ts.snap

```diff
@@ -0,0 +1,342 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`getRuleInclusions matches snapshotted list of production rules 1`] = `
+Object {
+  "accesskeys": Object {
+    "reason": "rule maps to BestPractice",
+    "status": "excluded",
+  },
+  "area-alt": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-allowed-attr": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-allowed-role": Object {
+    "reason": "rule maps to BestPractice",
+    "status": "excluded",
+  },
+  "aria-hidden-body": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-hidden-focus": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-input-field-name": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-required-attr": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-required-children": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-required-parent": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-roledescription": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-roles": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-toggle-field-name": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-valid-attr": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "aria-valid-attr-value": Object {
+    "reason": null,
+    "status": "included",
+  },
+  "audio-caption": Object {
+    "reason": null,
```

Comment:
`audio-caption` is disabled in axe. this gets pulled in because of https://github.com/microsoft/accessibility-insights-web/blame/6fd2816817db760818b35a3e711c02df3cc90c97/src/scanner/axe-rule-overrides.ts I'm exploring if these overrides are still relevant. If they are, we may be able to pull them in here so there's a reason for its explicit inclusion as well.